arch/arm/src/tlsr82/Make.defs

```diff
@@ -0,0 +1,87 @@
+############################################################################
+# arch/tc32/src/tlsr82/Make.defs
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.  The
+# ASF licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+############################################################################
+
+# Specify our HEAD assembly file.  This will be linked as
+# the first object file, so it will appear at address 0
+HEAD_ASRC  = cstartup_flash.S
+
+# Arch Flags
+AFLAGS +=-DMCU_STARTUP_FLASH
+
+# Common files in arch/arm/common
+CMN_CSRCS += arm_initialize.c arm_interruptcontext.c arm_idle.c
```

Comment:
let's use include common Makefile, e.g.:
common/Make.defs or armv6-m/Make.defs
Please reference PR https://github.com/apache/incubator-nuttx/pull/6165

Reply:
I'm not sure if that can be used due to
```
ifeq ($(CONFIG_SCHED_BACKTRACE),y)
CHIP_CSRCS += tc32_backtrace.c
endif
```

Reply:
fileter-out can help on this, it's already used in several place.

Reply:
@xiaoxiang781216 Done